packages/queued-request-controller/src/index.ts

```diff
@@ -1,16 +1,3 @@
-export type {
-  QueuedRequestControllerState,
-  QueuedRequestControllerCountChangedEvent,
-  QueuedRequestControllerEnqueueRequestAction,
-  QueuedRequestControllerEvents,
-  QueuedRequestControllerActions,
-  QueuedRequestControllerMessenger,
-  QueuedRequestControllerOptions,
-} from './QueuedRequestController';
-export {
-  QueuedRequestControllerActionTypes,
-  QueuedRequestControllerEventTypes,
-  QueuedRequestController,
-} from './QueuedRequestController';
+export * from './QueuedRequestController';
```

Comment:
Looks like this would export `controllerName` and `AllowedActions` at the package-level.

Reply:
Good catch, that was a mistake

Reply:
Fixed here: https://github.com/MetaMask/core/pull/3781/commits/5a95122eb11a33c087fdc4f9df8213e3aa656b38